src/components/directory/DirectoryPage.jsx

```diff
@@ -0,0 +1,123 @@
+    width: '100%',
+    overflowX: 'auto',
+  },
+  table: {
+    minWidth: 650,
+  },
+});
+
+class DirectoryPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: 0
+    };
+  }
+
+  render() {
+    const { classes } = this.props;
+
+    function handleChange(e, value) {
+      this.setState({ value: value });
+    }
+
+    function createData(name, job, username, password) {
+      return { name, job, username, password };
+    }
+
+    function Directory(props) {
+      const { label, rows } = props
+      return <Paper className={classes.directory}>
+        <Table className={classes.table} aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <TableCell>{label} Name</TableCell>
+              <TableCell>{label === 'Candidate' ? 'University Name' : 'Job Title'}</TableCell>
+              <TableCell>User Name</TableCell>
+              <TableCell>Password</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {rows.map(row => (
+              <TableRow key={row.name}>
+                <TableCell component="th" scope="row">
+                  {row.name}
+                </TableCell>
+                <TableCell>{row.job}</TableCell>
+                <TableCell>{row.username}</TableCell>
+                <TableCell>&bull;&bull;&bull;&bull;&bull;&bull;&bull;&bull;</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </Paper>;
+    }
+
+    const candidates = [
+      createData('Chris Evans', 'University of British Columbia', 'c.evans@gmail.com', '********'),
+      createData('Robert Downey Jr.', 'Simon Fraser University', 'r.downey@gmail.com', '********'),
+      createData('Scarlet Johansen', 'British Columbia Institue of Technology', 's.johansen@gmail.com', '********'),
+      createData('Tom Holland', 'University of Waterloo', 't.holland@gmail.com', '********'),
+      createData('Benedict Cumberbatch', 'University of Victoria', 'b.cumberbatch@gmail.com', '********'),
+    ];
+
+    const employees = [
+      createData('Captain America', 'Senior Project Manager', 'c.america@galvanize.com', '********'),
+      createData('Iron man', 'UX Designer', 'i.man@galvanize.com', '********'),
+      createData('Black Widow', 'Junior Software Developer', 'b.widow@galvanize.com', '********'),
+      createData('Spider man', 'Business Analyst', 's.man@galvanize.com', '********'),
+      createData('Dr. Strange', 'Senior Project Owner', 'd.strange@galvanize.com', '********'),
+    ];
+
+    return <div>
+      <div className={classes.header}>
+        <h1 className={classes.title}>Directory</h1>
+      </div>
+      <Paper style={{ margin: '40px 10px auto' }} square>
+        <Tabs
+          value={this.state.value}
+          indicatorColor="primary"
+          textColor="primary"
+          onChange={handleChange.bind(this)}
+          aria-label="disabled tabs example"
```

Comment:
I think we can remove these example code...?

Also, I'm not too sure how does `handleChange.bind(this)` work, I normally see people binding it in the constructor. Anything with `this` just gives me a headache haha. My way of doing this is define `handleChange = () => { }` (outside of the render function) and when passing it as a handler, either do `() => this.handleChange()` or just simply `this.handleChange`

Reply:
When I did `this.handleChange`, I got `this` is not defined error for some reason. This is my first time working with React, so I may not be aligned to the standard norm, so we could definitely discuss this in person on Tuesday!

Reply:
In the constructor you have to do something like this:
this.handleChange = this.handleChange.bind(this);

but your way works just fine, so I don't think we need to change it.
but if you want to pass any parameters then you have to do the arrow function way (onClick={() => this.handleChange(param)})

Reply:
Thanks for the insight Stefan!

I think it's because you defined handleChange within render function, so technically `this` does not contain the method handleChange